### PR TITLE
add dockerfile

### PR DESCRIPTION
**Pull request by cosunae**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:latest
+
```
Comment (twicki) [inline Dockerfile]:
```suggestion
LABEL maintainer="Carlos Osuna <Carlos.Osuna@meteoswiss.ch>"
LABEL description="Dawn build stage"
```

**Comment by twicki** [comment]
(test if jenkins is working)

**Comment by twicki** [comment]
launch jenkins

**Comment by cosunae** [comment]
launch jenkins

**Comment by cosunae** [comment]
@twicki  can you check this again?

**Comment by twicki** [comment]
closed in favor of #303

**Pull request closed without merging**